WebDevStudios/wd_s
# Hotfix/slick carousel copy

### DESCRIPTION ###
- Bumps version numbers in `package.json`
- Copies slick-carousel from `node_modules` into a theme directory on `npm run build` so the files are available in the theme (since we don't deploy `node_modules`).

### SCREENSHOTS ###
![folder directory](https://d.pr/free/i/6EnzQG+)
![updated scripts](https://d.pr/free/i/pLU61i+)

### OTHER ###

### STEPS TO VERIFY ###
1. Check out the branch
2. Run `npm install`
3. Run `npm run build`

### DOCUMENTATION ###
Will this pull request require updating the wd_s [wiki](https://github.com/WebDevStudios/wd_s/wiki)?

We don't have documentation around `package.json`, so I don't think this requires an update unless we want to start documenting that file. Maybe document the process to copy something from `node_modules` to the `assets` directory in the event that this is needed for more packages? Until we find a better solution, at least.
